Zend/tests/abstract_inheritance.phpt

```diff
@@ -0,0 +1,13 @@
+--TEST--
+Allow abstract function override
+--FILE--
+<?php
+
+abstract class A           { abstract function bar($x); }
+abstract class B extends A { abstract function bar($x); }
```

Comment:
Maybe add two more test for the same with the signature being changed (in a compatible manner and an incompatible manner)?